jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -43,6 +44,14 @@ static War getWarTask(Project project) {
     if (warPluginConvention == null) {
       return null;
     }
+
+    if (project.getPlugins().hasPlugin("org.springframework.boot")) {
+      Task bootWar = project.getTasks().findByName("bootWar");
```

Comment:
How does this handle users who are building jars with the springboot plugin?

Reply:
It doesn't.

- Jib already works well for "normal" spring boot java applications
- For "normal" spring boot java applications both `warPluginConvention` and `project.getTasks().findByName("bootWar")` would be null, so my proposed change wouldn't do anything in this case.

Reply:
Ah I see, I didn't realize that's how `bootwar` works. So yeah we'll take a look at this.